extensions/douban/src/search-music.tsx

```diff
@@ -116,6 +117,11 @@ export default function Command() {
                   shortcut={{ modifiers: ["cmd", "shift"], key: "d" }}
                   onAction={() => setShowingDetail(!showingDetail)}
                 />
+                <Action.CopyToClipboard
+                  title="Copy As Markdown"
+                  content={`[${music.title}(豆瓣)](${music.url})`}
```

Comment:
At the moment, Raycast doesn't support localization and only supports US English. Therefore, please avoid introducing your custom way to localize your extension.